clang/lib/Headers/feature-availability.h

```diff
@@ -10,24 +10,8 @@
 #ifndef __FEATURE_AVAILABILITY_H
 #define __FEATURE_AVAILABILITY_H
 
-#include <stdint.h>
+// feature-availability.h is deprecated - use availability_domain.h instead.
```

Comment:
Should we turn this comment into a real warning using `#warning`?

Reply:
I'd rather have a chance to move adopters off of it before it becomes a warning (although once adopters move off of it we should probably remove it entirely).